config/server.json

```diff
@@ -8,5 +8,9 @@
   },
   "mongo": {
     "connectionString": "mongodb://localhost/streams"
+  },
+  "schemaVersion": {
+    "gte": 0,
```

Comment:
yeah true - I was mapping it to the mongo query :) I will change that and then merge